python/lsst/dax/imgserv/imageREST_v0.py

```diff
@@ -93,27 +93,82 @@ def checkRaDec(raIn, decIn):
 def getRaw():
     return _getIFull(request, W13RawDb)
 
+# this will handle something like:
+# GET /image/v0/raw/ids?run=5646&camcol=4&field=694&filter=g
+@imageREST.route('/raw/ids', methods=['GET'])
```

Comment:
A break between the API and the implementation may not be a bad thing. Most of these queries are about 80% the same, following the calexp pattern or deepCoadd pattern, just accessing different repositories and policies.